src/libs/getDomainFromEmail.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'underscore';
+
+/**
+ * Checks if the expiresAt date of a user's ban is before right now
+ *
+ * @param {String} email
+ * @returns {String}
+ */
+function getDomainFromEmail(email) {
```

Comment:
No need for this – use `Str.extractEmailDomain` from expensify-common.

Reply:
Oooh didn't know about this... TIL!